README.md

```diff
@@ -361,7 +363,7 @@ Otherwise, if `protocol.webdav.sharedSecret` is not empty, the receiver MUST pas
 
 In both cases, when the Resource is a folder and the Receiving Server accesses a resource within that shared folder, it SHOULD append its relative path to that URL.
 
-Additionally, if `protocol.<protocolname>.permissions` include `mfa-enforced`, the Receiving Server MUST ensure that the Receiving Party has been authenticated with MFA.
+Additionally, if `protocol.<protocolname>.webdav.requirements` include `mfa-enforced`, the Receiving Server MUST ensure that the Receiving Party has been authenticated with MFA.
```

Comment:
If the variable `<protocolname>` has the value `'webdav' then it's already there, right?

Reply:
I think this was a logical error on my part, and a misunderstanding on how the example in the spec about how single protocol legacy/ multi protocol was worded. Should be fixed now.